dbms/src/Flash/Mpp/MPPTaskManager.cpp

```diff
@@ -31,11 +34,46 @@ extern const char random_task_manager_find_task_failure_failpoint[];
 extern const char pause_before_register_non_root_mpp_task[];
 } // namespace FailPoints
 
+namespace
+{
+// Print log for MPPTask which hasn't been removed for over 5 minutes.
+String getLongLiveMPPTasks(const std::unordered_map<String, Stopwatch> & monitored_tasks)
+{
+    String log_info;
+    for (const auto & iter : monitored_tasks)
+    {
+        auto alive_time = iter.second.elapsedSeconds();
+        if (alive_time >= 300)
+            log_info = fmt::format("{} <MPPTask is alive for {} secs, {}>", log_info, alive_time, iter.first);
```

Comment:
how about move a step forward, just return vector<pair<string, time>>, and do formating outside too?

Reply:
> how about move a step forward, just return vector<pair<string, time>>, and do formating outside too?

okk